README.md

```diff
@@ -29,3 +29,7 @@ that was received from auth service.
 1. To install dependencies run: `docker-compose exec verify yarn`.
 1. To run tests run: `docker-compose exec verify yarn test`.
 1. To build production image run `docker-compose -f docker-compose.prod.yml build --no-cache`.
+
+#How to generate docs?
```

Comment:
Fix it, pls. And 24 line too.

Reply:
@Aleserche Please clarify what to fix here?

Reply:
Add a space after `#`. To be rendered correctly.

Reply:
@Aleserche Correct, will fix in next PR, thanks!